### DISCUSSION TITLE
Meshery UI Figma Prototypes 🎨

### DISCUSSION THREAD

**Discussion by adithyaakrishna** [meshery/meshery#4991] [General]
### Meshery UI - Provider Login Prototype

https://user-images.githubusercontent.com/23498248/151198226-4e440585-09ff-4dd7-83a4-13e022c5b3d4.mp4

**Prototype Link:** https://www.figma.com/proto/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4275%3A28582&scaling=scale-down&page-id=347%3A6794&starting-point-node-id=4275%3A28582&show-proto-sidebar=1

### Meshery UI - Notification Login Prototype

https://user-images.githubusercontent.com/23498248/151198558-048b498f-d1f0-4f90-a994-ea3eb92cfbca.mp4

**Prototype Link:** https://www.figma.com/proto/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=4275%3A28013&scaling=scale-down&page-id=347%3A6794&starting-point-node-id=4275%3A28013&show-proto-sidebar=1

**Comment by leecalcote** [comment]
Be sure to extend the length of the Notification Center to the bottom of the screen. ✔️

**Reply by adithyaakrishna** [reply]
Done 💯

**Comment by leecalcote** [comment]
Don't forget to show notifications based on severity when clicking the severity filters.

**Reply by adithyaakrishna** [reply]
Done 💯

**Comment by leecalcote** [comment]
It'd be great to see the chevron on an individual notification bar active so that any users can see how the notification details will be formatted.
